kahuna/public/js/search/index.js

```diff
@@ -180,6 +180,77 @@ search.config(['$stateProvider', '$urlMatcherFactoryProvider',
             }]
         },
         views: {
+            selectionActions: {
+                template: `
+                <gr-top-bar
```

Comment:
I was going to do this once we were good on actually thinking it was a good idea.